content/upcoming/enry.md

```diff
@@ -0,0 +1,183 @@
+* [vendor.yml](https://github.com/github/linguist/blob/master/lib/linguist/vendor.yml)
+* [documentation.yml](https://github.com/github/linguist/blob/master/lib/linguist/documentation.yml)
+
+These files are parsed to retrieve the necessary information. Then the source files in *enry*  that offer this information to the rest of the project are generated and encapsulated as an internal subpackage [data](https://github.com/src-d/enry/tree/master/data).
+
+The whole process is automated and you only need to run `go generate` from the project's root directory to launch it. It allows *enry* to get updated without complex modifications when *linguist* adds new information.
+
+## Does enry actually have better performance?
+*enry*'s language detection has been compared with *linguist*.  In order to do that, linguist's project directory [linguist/samples](https://github.com/github/linguist/tree/master/samples) was used as a set of files to run benchmarks against.
+
+The number of language detections for each file in samples directory and per each time interval in a logarithmic scale has been measured for both tools, getting the following results:
+```
+enry processed files: 1839
+     1us-10us 5.220228%
+     10us-100us 67.645459%
+     100us-1ms 17.455139%
+     1ms-10ms 7.340946%
+     10ms-100ms 2.338227%
+
+linguist processed files: 1839
+     1us-10us 0.000000%
+     10us-100us 4.023926%r
+
+     100us-1ms 50.027189%
+     1ms-10ms 42.849375%
+     10ms-100ms 3.099511%
+```
+As you can see, *enry* was able to detect 67% of files in a time between 1us and 10us, while the majority of the files *linguist* processed are shifted to greater time intervals.
+
+Calculating the mean spent time to process a file for both tools, on average *enry* is 211% faster than *linguist*.
+
+## enry CLI
+*enry* can be used as a command too
+```
+$ enry --help
+enry, A simple (and faster) implementation of github/linguist
+usage: enry <path>
+       enry [-json] [-breakdown] <path>
+       enry [-json] [-breakdown]
+```
+, and it's programmed to return a similar output to *linguist*'s output.
+```
+$ enry
+11.11%    Gnuplot
+22.22%    Ruby
+55.56%    Shell
+11.11%    Go
+```
+The command has flags to get the results broken down by file,
+```
+$ enry --breakdown
+11.11%    Gnuplot
+22.22%    Ruby
+55.56%    Shell
+11.11%    Go
+
+Gnuplot
+plot-histogram.gp
+
+Ruby
+linguist-samples.rb
+linguist-total.rb
+
+Shell
+parse.sh
+plot-histogram.sh
+run-benchmark.sh
+run-slow-benchmark.sh
+run.sh
+
+Go
+parser/main.go
+```
+and to show it in JSON format,
+```
+$ enry --json
+{"Gnuplot":["plot-histogram.gp"],"Go":["parser/main.go"],"Ruby":["linguist-samples.rb","linguist-total.rb"],"Shell":["parse.sh","plot-histogram.sh","run-benchmark.sh","run-slow-benchmark.sh","run.sh"]}
+```
+
+The main difference with linguist's command is that enry doesn't need a git repository in the directory to analyse!
```

Comment:
That is important and might be worth highlighting in the `requirements` section [at the beginning](https://github.com/src-d/blog/pull/108/files#r125597628)